config/jenkins_infra.ci.jenkins.io.yaml

```
@@ -288,11 +288,15 @@ controller:
                     EOF
                     systemctl daemon-reload
                     systemctl restart docker
+                    export DEFAULT_JDK=/opt/jdk-11
+                    update-alternatives --install /usr/bin/java java "${DEFAULT_JDK}/bin/java" 2000
```

```suggestion
                    update-alternatives --install /usr/bin/java java "^${DEFAULT_JDK}/bin/java" 2000
```

Need to escape `$` sign on JCasc if you want it to be litteral